lib/top.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:modal_bottom_sheet/modal_bottom_sheet.dart';
+import 'package:retrospective_tool/input_form.dart';
+import 'package:retrospective_tool/widgets/kpt_note_widget.dart';
+
+import 'model/kpt_note.dart';
+import 'state/kpt_note_list.state.dart';
+
+class Top extends StatelessWidget {
+  const Top({Key? key}) : super(key: key);
+
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: const MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends HookConsumerWidget {
```

Comment:
MyHomePageって名前は変えた方が良いかもね（次の修正時に変えておきます）

Reply:
こっちがTopだったすね。。。一緒に全部写しちゃってました
```suggestion
class Top extends HookConsumerWidget {
```

この辺はmain.dartにいれたままでよかったかも。。
```dart
class Top extends StatelessWidget {
   const Top({Key? key}) : super(key: key);

   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
       title: 'Flutter Demo',
```